src/openrct2/ride/RideData.cpp

```diff
@@ -356,3 +354,17 @@ ResearchCategory RideTypeDescriptor::GetResearchCategory() const
     log_error("Cannot get Research Category of invalid RideCategory");
     return ResearchCategory::Transport;
 }
+
+static RideTrackGroup _enabledRidePieces = {};
+
+// used by RideConstruction.cpp and _legacy.cpp
```

Comment:
Not sure this comment is helping anyone really you can remove that as well, no one is gonna track where and how certain functions are used.